corehq/apps/cloudcare/static/cloudcare/js/formplayer/layout/views/progress_bar.js

```diff
@@ -23,6 +23,12 @@ hqDefine("cloudcare/js/formplayer/layout/views/progress_bar", function () {
                 $(this.el).find('.progress').addClass("hide");
                 $(this.el).find('.js-loading').removeClass("hide");
             } else {
+                if (hqImport('hqwebapp/js/toggles').toggleEnabled('USE_PROMINENT_PROGRESS_BAR')) {
```

Comment:
I am not quite sure why these styles have to be set for every time the progress is updated. Could you not just do it on the initial one?

Reply:
you're right. I overlooked that this was being done repeatedly.